### ISSUE TITLE
Вы автор?

### ISSUE THREAD

**Issue by Tomat7** [NikolayDikiy/ASOLED-by-andriano#2]
Николай, уточните пожалуйста - Вы автор библиотеки и первого поста в ветке http://arduino.ru/forum/proekty/asoled-kompaktnaya-biblioteka-dlya-oled-displeya-128kh64-s-kirillitsei-utf-8 ?

**Comment by NikolayDikiy** [comment]
Нет. Я скопировал её к себе для удобства использования в своих проектах.
